.github/workflows/test.yml

```diff
@@ -39,12 +64,23 @@ jobs:
         with:
           path: .venv
           key: venv-${{ matrix.python-version }}-${{ matrix.django-version }}-${{ hashFiles('poetry.lock') }}
+
       - name: Install dependencies
         run: |
           source "$HOME/.poetry/env"
           poetry config virtualenvs.in-project true
           poetry install
-          pip install -U "${{ matrix.django-version }}"
+          poetry run pip install \
+            --upgrade \
+          "${{ matrix.django-version }}" \
+          ${{ matrix.additional-dependencies }}
+
+      - name: Pull and build docker-compose services
+        if: ${{ matrix.services }}
+        run: |
+          docker-compose pull ${{ matrix.services }}
+          docker-compose up --detach ${{ matrix.services }}
+          # TODO: wait with some predefined timeout for services
```

Comment:
We can use `wait-for-it` or similar, see in our django template

Reply:
sure I will check it, for now, I was thinking about [`dockerize wait`](https://github.com/jwilder/dockerize)

Reply:
Yes, it is the same thing!